index.js

```diff
@@ -66,6 +74,10 @@ async function map (iterator, func, n = 16) {
   return results
 }
 
+async function forEach (iterator, func, n = 16) {
+  await map(iterator, func, n)
```

Comment:
Unfortunately this is not equivalent as it will keep accumulating the results in memory. Can you just use `mapIterator` and not accumulate the results?

Reply:
Fixed